remix-astwalker/src/sourceMappings.ts

```diff
@@ -1,28 +1,60 @@
 import { isAstNode, AstWalker } from './astWalker';
-import { AstNode, Location } from "./types";
+import { AstNode, LineColPosition, LineColRange, Location } from "./types";
+const util = require("remix-lib").util;
 
 export declare interface SourceMappings {
   new(): SourceMappings;
 }
 
 /**
- * Break out fields of an AST's "src" attribute string (s:l:f)
- * into its "start", "length", and "file index" components.
+ * Turn an character offset into a LineColPosition
+ *
+ * @param {Number} offset  - the character offset to convert.
```

Comment:
tsdoc recommends not to specify the type `{Number}` in comments for @params and @returns as it's already defined in typescript's type. See [documentation]'https://typedoc.org/guides/doccomments/) about tsdoc.

Reply:
Thanks for the link. I think I have everything converted over to the typedoc.org format.